README.md

```diff
@@ -1,85 +1,23 @@
-tinydecred
-==========
+# tinydecred
 
 [![Build Status](https://github.com/decred/tinydecred/workflows/Build%20and%20Test/badge.svg)](https://github.com/decred/tinydecred/actions)
-[![ISC License](https://img.shields.io/badge/license-ISC-blue.svg)](http://copyfree.org)
+[![ISC License](https://img.shields.io/badge/license-ISC-blue.svg)](https://copyfree.org/)
 
-## Overview
+TinyDecred is a Python 3 toolkit that can be used to integrate
+[Decred](https://decred.org/) into Python projects.
 
-A Python 3 Decred toolkit. The modules in TinyDecred can be used to integrate
-Decred into Python projects. Everything needed to create wallets to send and
-receive DCR.
+The [`decred`](./decred) package contains everything needed to create wallets
+to send and receive DCR.
 
-## Features
+The [`tinywallet`](./tinywallet) package contains such a light wallet based on
```

Comment:
## Is this phrase grammatically correct?

I'm thinking it could be `package contains a light wallet based on` without the `such`.

Reply:
Indeed, thanks.

Reply:
> The tinywallet package contains a wallet based on the **decred** toolkit

Reply:
Changed.